src/css/pagination.css

```diff
@@ -93,7 +93,7 @@
 
       path {
         fill: var(--dark);
-        fill-opacity: 0.5;
+        opacity: 0.5;
```

Comment:
![image](https://github.com/olexandrd/goit-advancedjs-team-3/assets/13851264/68c67020-d37a-4ead-aa21-355c459ef874)

Please check the error from validator

Reply:
<img width="1383" alt="image" src="https://github.com/olexandrd/goit-advancedjs-team-3/assets/13751198/7f8072dc-d396-4ef0-b5f4-429f8d151508">